src/routes.php

```diff
@@ -8,6 +8,11 @@
 // e.g route '/item/edit?id=1' will execute $itemController->edit(1)
 return [
     '' => ['HomeController', 'index',],
+    'sign-in' => ['SigninController', 'signin',],
+    'sign-up' => ['SignupController', 'signup',],
+    'hostpage' => ['HostpageController', 'hostpage',],
+    'bandpage' => ['BandpageController', 'bandpage',],
+    'sign-in/account' => ['AccountController', 'account',],
```

Comment:
pas besoin de préfixer la page profil par "sign-in" => account uniquement

Reply:
pour les pages hostpage, préférer l'url host, idem pour bandpage et penser à modifier le nom des méthodes